### DISCUSSION TITLE
what I shoud do for these translation type "vanished", "unfinished"?

### DISCUSSION THREAD

**Discussion by ghost** [jamulussoftware/jamulus#2683] [Technical Support & Help]
Hi, I am translating on Korean now, but I met some issue about these two types.

translation type="vanished" 
translation type="unfinished" 
some of them are translated (mostly type="vanished"), but all "unfinished" type is not translated and left it as blank.

Should I leave it as blank like others?
or translate and leave it? 
or change _translation type="unfinished"_ to this _translation_ after translating?

What should I do?

**Comment by ghost** [comment]
Well, I'm translating all now and will leave it as it is.

**Reply by pljones** [reply]
I believe `type="vanished"` means the text "upstream" that was originally requiring translation no longer exists.  So I think it happens when, for example, someone rewrites a string, maybe to make the English easier to understand.  That means the tool that generates the translations can no longer find the original value - so it's "vanished".  In the case you were making an update, you'd find a new, untranslated string had appeared.

Again, as I understand it, `type="unfinished"` means there's no translation - so the string needs translating for the language support to be complete.

**Reply by ghost** [reply]
Thanks :)
As soon as possible, let me PR it(Creating Korean translation_ko_KR.ts or Updating Korean translation_ko_KR.ts).

**Reply by ann0see** [reply]
The translation process will probably start very soon. There's still a little bit to change translation wise but after that you should be good to go.

**Reply by ghost** [reply]
Just now, I have done all the translation for Korean.
I hope it to be merged this version first.

**Comment by ann0see** [comment]
You should also get in touch with @chanjin65 to not double work.

**Reply by ghost** [reply]
Don't worry, we know each other well.

**Comment by ghost** [comment]
and, I think there should be Korean in language select in sever(translation_ko_KR.ts like file).
Can someone add Korean here?

 " message
        location filename="../util.cpp" line="696"/
        sourceFrench/source
        translation프랑스어/translation
    /message"
